sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CommunityGalleryImage.cs

```diff
@@ -71,6 +75,8 @@ public CommunityGalleryImage(OperatingSystemTypes osType, OperatingSystemStateTy
             Features = features;
             PurchasePlan = purchasePlan;
             Architecture = architecture;
+            PrivacyStatementUri = privacyStatementUri;
```

Comment:
Where are the .Net SDK test(s) for this feature?

Reply:
test added, also add new recorded test